### ISSUE TITLE
On iOS, sharing to Facebook shows "an error occurred please try again later"

### ISSUE THREAD

**Issue by alariccole** [eternalstorms/ESSVideoShare-for-OS-X-Lion#2]
This is shown in the web view, but once "Okay" is clicked the sign in prompt appears.

**Comment by eternalstorms** [comment]
weird. just tried it here on iPhone and iPad, the sign in prompt appears at once… Maybe facebook had some issues?

**Comment by eternalstorms** [comment]
are you still having this issue?

**Comment by alariccole** [comment]
Yes, it still occurs for me every time. I've tested on a device and the simulator, with 5.1 and 6.0.

**Comment by eternalstorms** [comment]
I'll check if facebook changed anything in their auth process.

**Comment by alariccole** [comment]
This is when using the essvideoshareiostest, I should mention. I just realized that perhaps the sample project doesn't have the latest, so maybe that's it.

**Comment by eternalstorms** [comment]
as a matter of fact, the sample codes currently do not include the most recent version of the source.
